### PR TITLE
Update share URL

### LINKED ISSUE #291: Functioning shortlink for app share
https://www.who.int/covid-19-app is the link for now

### PR DESCRIPTION
**Pull request by hspinks**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here: #291 
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] ~Included any relevant screenshots of UI updates.~
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?

Uses the new trampoline URL in share functionality: https://whocoronavirus.org/app

Closes #291 

## Did you add any dependencies?

N/A

## How did you test the change?

N/A

### CODE REVIEW

**Comment by hspinks** [comment]
@patniemeyer it looks like my auto-generating the intl stuff also updated a bunch of other strings, not sure if that's just because nobody else had run the auto-generation since those strings had changed

File: client/flutter/lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -55,7 +55,7 @@ class MessageLookup extends MessageLookupByLibrary {
     "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
     "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
     "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
-    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID019) press briefings including videos, audio and transcripts."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```
Comment (advayDev1) [inline client/flutter/lib/generated/intl/messages_en.dart]:
```suggestion
    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
```

File: client/flutter/lib/generated/intl/messages_en_GB.dart [hunk 2]
```diff
@@ -32,14 +32,31 @@ class MessageLookup extends MessageLookupByLibrary {
...
+    "homePagePageSliverListSettingsDataCollection" : MessageLookupByLibrary.simpleMessage("Allow WHO to collect data to better inform the public about Coronavirus updates. (need copy)"),
+    "homePagePageSliverListSettingsHeader1" : MessageLookupByLibrary.simpleMessage("Data Collection"),
+    "homePagePageSliverListSettingsHeader2" : MessageLookupByLibrary.simpleMessage("Language"),
     "homePagePageSliverListShareTheApp" : MessageLookupByLibrary.simpleMessage("Share The App"),
+    "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
+    "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
+    "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```
Comment (advayDev1) [inline client/flutter/lib/generated/intl/messages_en_GB.dart]:
```suggestion
    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
```

File: client/flutter/lib/generated/intl/messages_en_US.dart [hunk 3]
```diff
@@ -55,7 +55,7 @@ class MessageLookup extends MessageLookupByLibrary {
     "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
     "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
     "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
-    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID019) press briefings including videos, audio and transcripts."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```
Comment (advayDev1) [inline client/flutter/lib/generated/intl/messages_en_US.dart]:
```suggestion
    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
```

File: client/flutter/lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (hspinks) [reply client/flutter/lib/generated/intl/messages_en.dart]: heads up @advayDev1 that these files are all generated by `flutter_intl`, meaning that without updating the source in the `.arb` files, the next time we run the generator, these changes will be overwritten... might be worth a separate PR to update the originals

**Pull request merged**